http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala

```diff
@@ -665,7 +666,27 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
     when {
       wsm.getControlledAzureResourceApi(any)(any)
     } thenReturn IO.pure(api)
-    (wsm, api)
+
+    // enumerate workspace resources
+    when {
+      resourceApi.enumerateResources(any, any, any, any, any)
+    } thenReturn {
+      val resourceList = new ArrayList[ResourceDescription]
+      val resourceDesc = new ResourceDescription()
+      resourceDesc.metadata(new ResourceMetadata().name("cromwellmetadata"))
+      resourceDesc.resourceAttributes(
+        new ResourceAttributesUnion().azureDatabase(
+          new AzureDatabaseAttributes().databaseName("cromwellmetadata_abcxyz")
```

Comment:
I don't see this `cromwellmetadata_abcxyz` being used anywhere else in testing. Is there any way to check our wiring through, eg that we are plumbing _this_ value from WDS into the appropriate helm chart value for the cromwell runner?

Reply:
from what I can tell, that kind of check is happening in the test classes for individual app installation implementations - in this case, `CromwellRunnerAppInstallSpec.scala` (see [here](https://github.com/DataBiosphere/leonardo/pull/3756/files#diff-abe5ada8e5f43528f489448e1cd0a263ea966e967a450bc111802ae8e55e406aR21))

I don't know if we have end-to-end coverage for this kind of plumbing, or if the changes to `CromwellRunnerAppInstallSpec` suffice...